lib/travis/config/heroku/url.rb

```diff
@@ -22,7 +22,18 @@ def to_h
             super.reject { |key, value| key == :database }.merge(vhost: vhost)
           end
         end
-        Amqps = Amqp
+
+        # The amqps:// protocol needs to also pass along that it has SSL enabled
+        # for adapters such as march_hare
+        class Amqps < Amqp
+          def ssl
```

Comment:
(re-posting this bit of review as GitHub is pretty weird about outdated diff comments)

Added a method here so that using `config.amqp.ssl` would work, but I'm not utilizing the underlying `Struct` like the `Amqp` class is with its alias. As I understand it, there's two options here since we can't change a struct:

1. I can add ssl to the base class in here. But then all other protocols would get that param too which would increase the risk/surface area of this change.
1. I can reimplement everything the base class and `Amqps` and make this class stand-alone, but then we're repeating most things.
Or this may not be an issue. And uses of this don't assume anything about the underlying `Struct` other than to create hashes, which is contained in this class.